counterpartylib/lib/api.py

```diff
@@ -764,6 +779,255 @@ def handle_root(args_path):
                 # Not found
                 return flask.Response(None, 404, mimetype='application/json')
 
+        @dispatcher.add_method
+        def sendrawtransaction(tx_hex):
```

Comment:
This call specifically will validate any bitcoin transaction, so it may be better to move it out of the micropayments module.